src/router/routes.js

```diff
@@ -334,48 +334,6 @@ export default [
 	{
 		path: '/styleguide',
 		component: () => import('@/pages/StyleGuide'),
-		children: [
```

Comment:
@ryan-ludwig Could would create a fallback route for `styleguide/*` to redirect back to just `/styleguide`?